src/include/main/db_main.h

```diff
@@ -229,17 +229,20 @@ class DBMain {
      * @param traffic_cop argument to the ConnectionHandleFactor
      * @param port argument to TerrierServer
      * @param connection_thread_count argument to TerrierServer
+     * @param use_unix_domain_sockets argument to TerrierServer
+     * @param socket_directory argument to TerrierServer
      */
     NetworkLayer(const common::ManagedPointer<common::DedicatedThreadRegistry> thread_registry,
                  const common::ManagedPointer<trafficcop::TrafficCop> traffic_cop, const uint16_t port,
-                 const uint16_t connection_thread_count) {
+                 const uint16_t connection_thread_count, const bool use_unix_domain_sockets,
```

Comment:
As a general style nit: We should prefer enums over a boolean arg here. It gives the option of easily adding another value to an enum like `SocketType` without refactoring the signature of the function.